shadows/playservices/src/main/java/org/robolectric/shadows/gms/ShadowGooglePlayServicesUtil.java

```diff
@@ -7,7 +7,7 @@
 import android.content.DialogInterface.OnCancelListener;
 import android.content.Intent;
 import android.content.res.Resources;
-import android.support.v4.app.Fragment;
+import androidx.fragment.app.Fragment;
```

Comment:
I think the problem is that if you make this change, it won't match the signature of the method used in https://mvnrepository.com/artifact/com.google.android.gms/play-services-base/8.4.0:


```
  public static boolean showErrorDialogFragment(int, android.app.Activity, android.support.v4.app.Fragment, int, android.content.DialogInterface$OnCancelListener);
```

Reply:
Is ShadowGooglePlayServicesUtil even used any more? Maybe we should deprecate it and delete it, similar to how I'd like to remove the supportv4 shadows.

Reply:
I updated `ShadowGooglePlayServicesUtil` against recent GooglePlayServicesUtil APIs.